pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -635,6 +657,7 @@ func Run(ps *pubsub.PubSub) {
 	duration := time.Duration(ctx.vdiskGCTime / 10)
 	ctx.gc = time.NewTicker(duration * time.Second)
 	ctx.gc.Stop()
+	go casBlobMonitor(&ctx)
```

Comment:
Are the casClient APIs safe to call from different goroutines? Are they just a wrapper around some socket send/receive?

Reply:
The routine interacts with conatinerd's socket. There is no shared data in the CAS package so multiple routines can interact with the package at the same time.

Reply:
OK

Reply:
OK